src/zmf.cpp

```diff
@@ -331,7 +331,7 @@ int main( int argc, char *argv[] )
     Debug( 1, "Got image, writing to %s", path );
 
     FILE *fd = 0;
-    if ( (fd = fopen( path, "w" )) < 0 )
+    if ( (fd = fopen( path, "w" )) < (void *)0 )
```

Comment:
I'm pretty sure this is incorrect... I will have to do some research to figure out what it should be.  I always thought that file descriptors were integers...